travis/prepare/prepare-javascript

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Configuring display
+sh -e /etc/init.d/xvfb start > /dev/null 2>&1
+export DISPLAY=:99
+
+# Installing google-chrome
+wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb > /dev/null 2>&1
+sudo dpkg -i google-chrome-stable_current_amd64.deb > /dev/null 2>&1
```

Comment:
Btw. we may come back to container-based infrastructure, if there was a possibility to install Google Chrome on that. There's a PR for that: https://github.com/travis-ci/apt-source-whitelist/issues/35